solution/ui/regulations/eregs-component-lib/src/components/shared-components/results-item-parts/SubjectChips.vue

```diff
@@ -12,7 +10,7 @@ const props = defineProps({
     },
 });
 
-const sortedSubjects = props.subjects.sort(sortSubjects);
+const sortedSubjects = props.subjects.toSorted(sortSubjects);
```

Comment:
linter caught this one; `sort` sorts in place (which we don't want) while `toSorted` creates a sorted copy.